scripts/create_addon

```diff
@@ -91,6 +91,13 @@ pack_addon() {
   if [ ! -z "$PKG_ADDON_NAME" ] ; then
     ADDON_NAME="$PKG_ADDON_NAME"
   fi
+  for f in $PKG_DIR/repo/screenshot-*.{jpg,png}; do if [ -f $f ];
```

Comment:
`skin.estuary` uses the `resources` folder for artwork assets (icons, fanart, screenshots): https://github.com/xbmc/xbmc/blob/master/addons/skin.estuary/addon.xml#L112-L123

Is there any reason not to do the same? I suppose there's the possibility of a naming clash, but then what's to say an addon might have a `repo` folder in future? In that case, should we use something very unlikely to clash, such as `.libreelec`?